### DISCUSSION TITLE
Use white color as the filling for the M20, M19, Mxx text

### DISCUSSION THREAD

**Discussion by ibiza240** [andrewgioia/keyrune#254] [General]
Hi!
First, thanks for the great library.

Would it be possible to use white instead of the container background color for the Mxx text?

For example:
![image](https://user-images.githubusercontent.com/6589646/81070252-f68c4e80-8eb0-11ea-85bd-001ff1e3252d.png)

When used over a red background, the text is red...it would be better to force white I think, regardless of the container background.

Thanks!

**Comment by andrewgioia** [comment]
Just an update here that this is definitely on the radar and as of 3.7.0 the 2 newest sets that do this are supported! You can add `.ss-duo` to those 2 symbols (currently just `.ss-xdnd` and `.ss-xin4`) and they'll use 2 separate glyphs to correctly create this print effect.

I plan to address the others either in periodic updates in v3.X as I have time, but definitely in the v4.0 overhaul.

**Comment by andrewgioia** [comment]
Moving this (and all older v4 issues) to discussions!
